ppr-ui/src/utils/validation-helper.ts

```diff
@@ -244,6 +244,10 @@ export function validateSearchRealTime (
       if (searchValue?.length > 70) {
         validation.searchValue.message = 'Maximum 70 characters'
       }
+      if (specialCharsLax.test(searchValue)) {
+        validation.searchValue.message =
+          "Names don't normally contain special characters"
```

Comment:
Single quotes here

Reply:
Other strings with single quotes in words have double quotes, so I made the same.

Reply:
Just to clarify, i didn't mean outside this file. 
Just ensure there is consistency in this file and we can tackle other instances as we go :)

Reply:
Excellent, thanks for the update!